lib/chef/provider/user/mac.rb

```diff
@@ -212,6 +202,16 @@ def create_user
             end
           end
 
+          # createhomedir needs user GID set first
+          # otherwise createhomedir will do nothing
+          # Always create homedir for all users
+          # because 'sysadminctl' does but 'dscl' does not
+          unless ::File.directory?(new_resource.home) && ::File.exist?(new_resource.home)
```

Comment:
why was the check for `if new_resource.manage_home` removed here?

Reply:
This is because `sysadminctl` will create user homedir (in most cases) while `dscl` will not. Removing this check will make this provider always create homedir when creating a user. In this way, downstream users will (most likely) not need to change how they use this provider.